### PR TITLE
Kevin us0 database

### PR DESCRIPTION
**Pull request by FranciscoKevin**
Create the entities and user mysql

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=100)
+     */
+    private string $email;
+
+    /**
+     * @ORM\Column(type="string", length=100)
+     */
+    private string $password;
```
Comment (LucyMerlier) [inline src/Entity/User.php]: password should be 255 characters, not 100

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
password length should be 255, not 100

**Review by LucyMerlier — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
You probably should a User entity implementing the UserInterface
You should probably avoid having too small database fields. 50 characters for names can be too small

**Review by LucyMerlier — DISMISSED** [review]

**Review by RomainClair — COMMENTED** [review]
You probably shouldn't have only nullable fields in all your entities.

**Review by LucyMerlier — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**